frontend/src/js/components/deployments/deployment-report/DeviceList.tsx

```diff
@@ -72,8 +72,8 @@ const deviceListColumns = [
     title: 'id',
     renderTitle: ({ idAttribute }) => idAttribute.attribute,
     render: ({ device, idAttribute }) => (
-      <Link style={{ fontWeight: 'initial', opacity: idAttribute.attribute === 'name' ? 0.6 : 1 }} to={`/devices?id=${device.id}`}>
-        <DeviceIdentityDisplay device={device} isEditable={false} />
+      <Link style={{ fontWeight: 'initial' }} to={`/devices?id=${device.id}`}>
+        {getDeviceIdentityText({ device, idAttribute })}
```

Comment:
was DeviceIdentityDisplay the issue?

Reply:
Not necessarily 🙈 - the combination of this also having to work with the name tag and not being editable - there's an exemption in the DeviceIdentityDisplay for the name tag - and I just took the shortcut of showing the rendered content instead of fixing the DeviceIdentityDisplay (which is used in several other places)... I'll fix that instead!